### PR TITLE
Include setup.py and MIT license

### PR DESCRIPTION
**Pull request by Ayiden**
Added MIT LICENSE and setup.py files. Swapped the license out to the MIT License since I figured it was more representative of this repos purpose. Let me know your thoughts

### CODE REVIEW

File: Python/setup.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+setup(
+    name='kaiju-quality-examples',
+    version='0.0.1',
+    packages=['utilities'],
+    url='https://github.com/eschlange/kaiju-quality-examples',
+    license=license,
+    classifiers=['Programming Language :: Python :: 3.10'],
+    author='Kaiju Quality Consulting LLC',
+    author_email='eric.schlange@pizzahut.com',
```
Comment (eschlange) [inline Python/setup.py]: PH reference

**Review by eschlange — CHANGES_REQUESTED** [review]
Reference to a soon defunct email address, otherwise looks good!

File: Python/setup.py (hunk 1 above)
Comment (Ayiden) [reply Python/setup.py]: Yeah, I pulled this instead of rebuilding. I just forgot to push that update. Just recommitted

Comment (Ayiden) [reply Python/setup.py]: My B 😢

**Review by eschlange — APPROVED** [review]

**Pull request merged**